fractor/src/Helper/StringUtility.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\Fractor\Helper;
+
+use Symfony\Component\String\UnicodeString;
```

Comment:
I guess we should make the dependency symfony/string explicit and define it in the composer.json